pdns/histogram.hh

```diff
@@ -120,7 +120,7 @@ public:
     return b <= bu.d_boundary;
   }
 
-  inline void operator()(uint64_t d)
+  inline void operator()(uint64_t d) const
```

Comment:
I don't see why you would want to make this method `const` It's a modifier, incrementing a bucket. If it's a regular method the `mutable` above can go to.
Same for the `submit*` methods and the `mutable in `ResponseStats`. Or am I missing something?

Reply:
I agree it's weird, the idea is that it guarantees that only the mutable parts can be modified, and that we can't somehow alter the vector of buckets, or the name of a bucket, which would not be thread-safe. But, yeah, it's counter-intuitive so I would understand if you don't like it :)

Reply:
OK, I now understand better. It still feels a bit weird, but otoh you make very clear only the counts can change after construction. So I think I'll take it.